cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/main/java/co/cask/cdap/security/authorization/sentry/binding/conf/AuthConf.java

```diff
@@ -37,6 +37,11 @@
   public static final String INSTANCE_NAME = "instance.name";
   // a comma separated list of users who will be superusers
   public static final String SUPERUSERS = "superusers";
+  // TTL for authorization cache entries
+  public static final String CACHE_TTL_SECS = "cache.ttl.secs";
+  // Maximum number of entries to keep in the cache. An entry of <= 0 would disable caching.
```

Comment:
This is according to Terence's suggestion here https://github.com/caskdata/cdap/pull/8750#discussion-diff-117072821R2719